processing/src/main/java/org/apache/druid/math/expr/BuiltInExprMacros.java

```diff
@@ -29,10 +29,24 @@
 
 public class BuiltInExprMacros
 {
+  public static final String COMPLEX_DECODE_BASE64_NAME = "complex_decode_base64";
+  public static final String COMPLEX_DECODE_BASE64_ALIAS_NAME = "decode_base64_complex";
```

Comment:
Do we plan to hold a set of built-in aliases for different macros here? In that case, we can keep a Map of <alias_name, actual_name> so that a new function can add to the map